person.rb

```diff
@@ -1,18 +1,15 @@
-require_relative 'nameable'
+class Person
+  attr_accessor :name, :rentals
 
-class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id
-
-  def initialize(age, name = 'Unknown', parent_permission: true)
-    super()
-    @id = Random.rand(1..1000)
+  def initialize(name)
     @name = name
-    @age = age
-    @parent_permission = parent_permission
```

Comment:
- [ ] Please kindly take note that you shouldn't had deleted the parameters that were initialized in the constructor, because this milestone is a continuation of previous ones. Thanks. 🙏